src/main/webapp/WEB-INF/view/profile.jsp

```diff
@@ -56,8 +83,21 @@ User user = (User) request.getAttribute("user");
     <hr/>
     <h1><%= username %>'s Sent Messages</h1>
 
+    <div id="chat">
+      <ul>
+        <% for (Message message : messagesByUser) {
+          Instant time = message.getCreationTime();
+          String creation = DateTimeFormatter.RFC_1123_DATE_TIME.withZone(ZoneOffset.UTC).format(time);
+        %>
+          <li><strong><%= creation %>:</strong> <%= message.getContent() %></li>
+          <% } %>
+        </ul>
+    </div>
+
+
```

Comment:
Maybe you should reuse the methods on `activity-helper.jsp`, specifically, `formatCreationTime()`  and `formatMessage()`. That way we could update the UI easier in the future.

Reply:
Agree, we definitely want to reuse these code since they should look the same.